src/less/components/button.less

```diff
@@ -285,119 +11,73 @@
     vertical-align: middle;
     touch-action: manipulation;
     cursor: pointer;
-    line-height: 1rem;
+    line-height: 23.72px;
     border: 1px solid transparent;
     white-space: nowrap;
     user-select: none;
-
-    /* padding-top-bottom, padding-left-right, font-size, line-height */
-    .button-size(@btn-padding-horizontal; @btn-default-height; @btn-font-size; @line-height-large; @btn-border-radius-large);
-    .button-icon-size(@btn-padding-horizontal; @btn-default-height; @btn-default-width; @btn-font-size; @line-height-large; @btn-border-radius-large);
-
-    .material-icons,
-    .material-icons-outlined {
-        font-size: 1.25rem;
-        position: relative;
-        display: inline-block;
-        line-height: 1;
-
-        &:not(:last-child) {
-            margin-right: 0.5rem;
-        }
-    }
-
-    &.btn-icon-xl {
-        .button-icon-size(0.45rem; 3rem; 3rem; @font-size-small; @line-height-small; @btn-border-radius-small);
-
-        .material-icons-outlined {
```

Comment:
`.material-icons-outlined` "font-size" declarations have been deleted. I am not sure it is supposed to be deleted (but I had previously no awareness we had specific font-size declarations for outlined)